app/reducers/article.js

```diff
@@ -12,26 +12,19 @@ function getInitialState() {
   return {
     article: {},
     currentArticle: null,
-    nextArticle: null,
-    topics: topicJsonMock.results
+    nextArticle: null
```

Comment:
There won't be a nextArticle, go ahead and remove.
There will be an endpoint for requesting an article to work on, and if we want to be slightly ambitions in the saveArticleHighlights endpoint, we could save a round-trip and return the next article to work on, provided there were no errors to report on the saving of the highlights.